lib/services/elasticsearch.js

```diff
@@ -206,7 +206,7 @@ class ElasticSearch extends Service {
 
     return this.client.get(esRequest)
       .then(result => {
-        if (result._source && 
+        if (result._source &&
```

Comment:
Should not be
```
if (
result._source
&& result._source._kuzzle_info
&& !result._source._kuzzle_info.active
&& !request.input.args.includeTrash
) {
```
?

Reply:
I didn't change this: the only change is the trailing space that has been removed by my editor.
This code predates the new wrapping rules, so it shouldn't be blocking this PR.